reference/SUPPORTED_QUERIES.md

```diff
@@ -0,0 +1,159 @@
+# EQL Supported Queries
+
+`eqlpy` supports querying on encrypted data through EQL's [specialized functions](https://github.com/cipherstash/encrypt-query-language#searching-data-with-eql).
+
+## Table of contents
+
+- [Query types](#query-types)
+- [Required index types](#required-index-types)
+- [Supported query interfaces](#supported-query-interfaces)
+  - [Django ORM](#django-orm)
+  - [SQLAlchemy](#sqlalchemy)
+  - [psycopg](#psycopg)
+
+## Query types
+
+The `eqlpy` library enables the following types of queries on encrypted data:
+
+- Equality where values are exactly matched, like `X = Y`
+- Partial matching where a string is a substring of another, similar to `X like Y`
+- Comparison where a value is greater than, less than, or equal to another, like `X > Y`
+  - Ordering is supported where comparison is supported, like `ORDER BY C`
+- JSONB containment where a document is contained in another, like `X @> Y`
+- JSONB field extraction where a value is extracted from a document with a path, like `$.X.Y`
+- JSONB field comparison where values from specific fields are compared, like `$.X > Y`
+- JSONB grouping where values are grouped by specific JSONB fields, like `GROUP BY $.X`
+
+## Required index types
+
+- Equality queries require either the `unique` or `ore` index type
+- Partial matching of strings requires the `match` index type
+- Comparison requires the `ore` index type
+- JSONB queries require the `ste_vec` index type
+
+## Supported query interfaces
+
+### Django ORM
+
+For Django ORM, `eqlpy.eqldjango` provides custom field types that allow querying similar to how Django queries are typically run:
+
+```python
+Customer.objects.get(name__match="carol")
+```
+
+The following table shows custom field types provided by `eqldjango`.
+
+| EncryptedValue subclass | Supported lookups                | Supported index type |
+|-------------------------|----------------------------------|----------------------|
+| EncryptedText           | eq (EncryptedUniqueEquals)       | "unique"             |
```

Comment:
Suggest putting anything that is a class name or "symbol" in backticks. `EncryptedText` instead of EncryptedText.

Reply:
`EncryptedText` also supports ORE but its only useful for ORDER BY. Maybe we could add that later because I think we don't yet support ordering in eqlpy, right?

Reply:
`ORDER BY` is supported ilke this: `Customer.objects.order_by(CsOre648V1(F("name")).desc()).values().all()` so we coulld add it but that may require further explanation. I don't know if we can make it simpler than that without doing a lot of work